pkg/content/content.go

```diff
@@ -93,12 +100,18 @@ func blobSize(bucket *bolt.Bucket) (int64, error) {
 
 // GC clean the local caches by cfg.Provider.GCPolicy configuration
 func (content *Content) GC(ctx context.Context) error {
+	//clear the intermediate file created in convert
```

Comment:
Why do we need to call GC here?

Reply:
To clear the temp file created by containerd convert.

Reply:
We will clear the temp file after each convert.